packages/fetchye/__tests__/computeKey.spec.js

```diff
@@ -58,4 +58,46 @@ describe('computeKey', () => {
     };
     expect(computeKey('uri', firstOptions).hash).toBe(computeKey('uri', secondOptions).hash);
   });
+
+  it('should return a different, stable hash, if the option mapKeyToCacheKey is passed', () => {
+    expect(computeKey(() => 'abcd', {
+      mapKeyToCacheKey: () => 'efgh',
+    })).toMatchInlineSnapshot(`
+      Object {
+        "hash": "a0e09d568bb5b47c046b0fac7a61ca10196151cc",
+        "key": "abcd",
```

Comment:
Should this be `efgh`?

Reply:
no, the 'key' is the 'url' passed ultimately to fetch client.

its not obvious because its hashed, but the hash field 'contains' the 'efgh' in its inputs, and no-longer contains the 'abcd' anywhere in its inputs.

the point is to only allow control of the 'cacheKey' which is invisible inside of computeKey as its incorporated into the hash.

Reply:
Can you write a test that makes this apparent?

Reply:
https://github.com/americanexpress/fetchye/pull/71/commits/ab34fdbf292afe245109bae87dc3eaac52f83884#diff-17514781228925397fe9760535d25fc9ad1aad4d3b608f68f10fb4469bd3fe88R82-R89

Added :)